File: Data Structures & Algorithms/design-hashset/submission-1.java
class MyHashSet {
    boolean[] falseArray = new boolean[1000001];//since 0 <= key <= 1,000,000 (using boolean array)
    public MyHashSet() {
    }
    
    public void add(int key) {
        falseArray[key] = true;
        }
    
    
    public void remove(int key) {
        falseArray[key] = false;   
    }
    
    public boolean contains(int key) {
        return falseArray[key];
    }    
}

/**
 * Your MyHashSet object will be instantiated and called as such:
 * MyHashSet obj = new MyHashSet();
 * obj.add(key);
 * obj.remove(key);
 * boolean param_3 = obj.contains(key);
 */

File: Data Structures & Algorithms/design-hashmap/submission-1.py
class Node:
    def __init__(self, key, value):
        self.key = key
        self.value = value

class MyHashMap:

    def __init__(self):
        self.map = []
        

    def put(self, key: int, value: int) -> None:
        for node in self.map:
            if node.key == key:
                node.value = value
                return
        self.map.append(Node(key, value))
            
        
    def get(self, key: int) -> int:
        for node in self.map:
            if node.key == key:
                return node.value 
        return -1
        

    def remove(self, key: int) -> None:
        for node in self.map:
            if node.key == key:
                self.map.remove(node)
                return
        


# Your MyHashMap object will be instantiated and called as such:
# obj = MyHashMap()
# obj.put(key,value)
# param_2 = obj.get(key)
# obj.remove(key)